Remove the default root window grey stipple pattern, replacing it with
blackness.

--- dix/window.c.orig
+++ dix/window.c
@@ -119,8 +119,8 @@
  *
  ******/
 
-static unsigned char _back_lsb[4] = {0x88, 0x22, 0x44, 0x11};
-static unsigned char _back_msb[4] = {0x11, 0x44, 0x22, 0x88};
+static unsigned char _back_lsb[4] = {0x00, 0x00, 0x00, 0x00};
+static unsigned char _back_msb[4] = {0x00, 0x00, 0x00, 0x00};
 
 int screenIsSaved = SCREEN_SAVER_OFF;
 
